scripts/instperf

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+
+import logging
+from subprocess import Popen, PIPE
+import time
+
+# grab the top five memory consuming processes, returning them in a string of
```

Comment:
Could you please extend the comment with information that we have about this script and its use cases?